qiskit_qec/utils/pauli_rep.py

```diff
@@ -1997,12 +2024,19 @@ def _encode_of_tensor_str(
     def _find_encode(t_str: str, ecode: bool):
```

Comment:
@dsvandet I can't find where we use `_find_encode`. What is it for?

Reply:
I see that it's called in line 1934 `return np.array([_find_encode(item) for item in phase_str])` to find the encoding one by one of a string.

Reply:
Thanks! My bad, when I looked at it, I was off by a tab and thought it was inside the original function, recursive.